csrc/compute_at_map.cpp

```diff
@@ -374,6 +374,7 @@ void IterDomainGraph::build(Fusion* fusion) {
       if (first_output_tv == nullptr) {
         first_output_tv = c_tv;
       } else {
+        if (ir_utils::hasUniformSiblings(expr)){
```

Comment:
Skip mapping multiple outputs of the expr to each other for `SdpaOps`.

The code at L438 wil map producers and consumers.